src/assets/content/dictionary/application-programming-interface-api.md

```diff
@@ -1,6 +1,11 @@
 ---
 title: Application Programming Interface (API)
-definition: ''
+definition: 'The word Application refers to any software with a distinct function, and interface can be thought as a contract of service between two applications.This contract defines how the two communicate with each other using requests and responses. 
+
+Then a API sets contracts with defined rules that explain how computers or applications communicate with one another.This allows services and products to communicate with each other and leverage each other's data and functionality through a documented interface.'.
```

Comment:
```suggestion
definition: "The word Application refers to any software with a distinct function, and interface can be thought as a contract of service between two applications. This contract defines how the two communicate with each other using requests and responses. 

Then a API sets contracts with defined rules that explain how computers or applications communicate with one another. This allows services and products to communicate with each other and leverage each other's data and functionality through a documented interface."
```
bug: Since you have a single quote `'` inside the text, you'll need to use double quotes `"` to wrap the whole definition. I've also suggested to add spaces between the sentences for clarity. You can accept the suggestion via the Github UI, or you can make an extra commit to this PR.

Reply:
Thank you for your suggestions, I commit them and added space for better readability.